packages/cid-verifier/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "cid-verifier",
+  "version": "1.0.0",
+  "description": "Verify that a CID's content is safe for the web",
+  "private": true,
+  "type": "module",
+  "main": "./dist/worker.mjs",
+  "scripts": {
+    "lint": "standard",
+    "build": "tsc && node scripts/cli.js build",
+    "predev": "node scripts/cli.js ipfs --start",
+    "dev": "miniflare dist/worker.mjs --watch --debug -m",
+    "postdev": "node scripts/cli.js ipfs --stop",
+    "test": "npm run test:setup && npm-run-all -p -r mock:ipfs.io mock:cf-ipfs.com test:worker",
+    "test:worker": "ava --verbose test/*.spec.js",
+    "test:setup": "npm run build",
+    "mock:ipfs.io": "smoke -p 9082 test/mocks/ipfs.io",
+    "mock:cf-ipfs.com": "smoke -p 9083 test/mocks/cf-ipfs.com"
```

Comment:
I know you did not get to this yet, but since I am here this should be `mock:google-api`

Reply:
yea, i'm still in active development on it